sw/device/silicon_creator/lib/crypto/tests/ecdsa_p256_functest.c

```diff
@@ -49,15 +50,22 @@ rom_error_t compute_digest(void) {
 rom_error_t sign_then_verify_test(void) {
   ecdsa_p256_signature_t signature;
   hardened_bool_t verificationResult;
+  otbn_error_t err;
 
   // Generate a signature for the message
   LOG_INFO("Signing...");
-  RETURN_IF_ERROR(ecdsa_p256_sign(&digest, &kPrivateKey, &signature));
+  err = ecdsa_p256_sign(&digest, &kPrivateKey, &signature);
+  if (err != kErrorOtbnOk) {
```

Comment:
Maybe a macro for this specific idiom is worthwhile:

```c
FOLD_OTBN_ERROR(ecdsa_p256_verify(...));
```

This also helps keep changes in the callsites to a minimum. WDYT?

Reply:
That makes sense; I'll implement it!